src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -99,14 +99,7 @@ public void run() {
             }
 
             // Fixes #2576 - Remove any deleted instances of BlockStorage
-            Iterator<BlockStorage> worlds = SlimefunPlugin.getRegistry().getWorlds().values().iterator();
-            while (worlds.hasNext()) {
-                BlockStorage storage = worlds.next();
-
-                if (storage.isMarkedForRemoval()) {
-                    worlds.remove();
-                }
-            }
+            SlimefunPlugin.getRegistry().getWorlds().values().removeIf(BlockStorage::isMarkedForRemoval);
```

Comment:
What is the performance on this?

Reply:
they do the exact same code. The only difference is removeIf obviously takes a Predicate so runs that. This way is just much cleaner

https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/Collection.java#L571-L582